### PR TITLE
Intropage tests

### PR DESCRIPTION
**Pull request by dinodsaurus**
<br/><br/><br/><url>LiveURL: https://kiwicomsmart-faq-hphmebkrrc.now.sh
</url>
</url>

### CODE REVIEW

File: src/IntroPage/IntroPage.stories.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import StoryRouter from 'storybook-router';
+
+import IntroPage from './';
+
+storiesOf('IntroPage', module)
+  .addDecorator(StoryRouter())
+  .add('Default', () => <IntroPage />);
```
Comment (jalbertsr) [inline src/IntroPage/IntroPage.stories.js]: I think you noticed that we almost never use storybook even having it installed (shame on us), we use it so little we haven't realized that storybook wasn't working until your (https://github.com/kiwicom/smart-faq/pull/210), but if we start using it more often I think we should use it to show components not "screens", even so there won't be many components since we relay in Orbit Components as ui library.  what do you think ? @jaroslav-kubicek @git-toni @kantord @edgarordonez

Comment (dinodsaurus) [reply src/IntroPage/IntroPage.stories.js]: Yep makes sense.. sorry for the trouble ;)

Comment (git-toni) [reply src/IntroPage/IntroPage.stories.js]: I totally agree with @jalbertsr . Storybook seems like a great way to showcase our more "complex" components such as Accordion, Breadcrumbs, etc.

File: src/helpers/defaultTestProps.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// @flow
```
Comment (jaroslav-kubicek) [inline src/helpers/defaultTestProps.js]: is this file necessary? it's used only once

if we need the same in multiple test files, let's hide it in some `__tests__` folder

File: src/IntroPage/__tests__/IntroPage.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+describe('IntroPage', () => {
+  const props = {
+    user,
+    history: {
+      push: jest.fn(),
+    },
+  };
+  it('should match snapshot', () => {
+    const wrapper = shallow(<PureIntro {...props} />);
+    expect(wrapper).toMatchSnapshot('<IntroPage />');
```
Comment (jaroslav-kubicek) [inline src/IntroPage/__tests__/IntroPage.test.js]: in case of IntroPage ,it  doesn't matter much, but in general, snapshots tests doesn't make sense for big components as they tend to change too often, snapshot is long & noone is paying attention to it.

small, leaf components => shapshots are fine
bigger components => verifying the existence of particular child component with enzyme `find` or simulation of the events usually works better than huge snapshot

File: src/IntroPage/IntroPage.stories.js (hunk 1 above)
Comment (jaroslav-kubicek) [reply src/IntroPage/IntroPage.stories.js]: I agree as well

File: src/helpers/defaultTestProps.js (hunk 2 above)
Comment (dinodsaurus) [reply src/helpers/defaultTestProps.js]: I would say this was a premature optimization 😅
